sdk/appconfiguration/Azure.Data.AppConfiguration/samples/README.md

```diff
@@ -17,7 +17,7 @@ description: Samples for the Azure.Data.AppConfiguration client library
  - [Read Revision History](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample4_ReadRevisionHistory.md)
  - [Get a Configuration Setting](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample5_GetSettingIfChanged.md)
  - [Update a Configuration If Unchanged](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample6_UpdateSettingIfUnchanged.md)
- - [Mock a client for testing using the Moq library](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample7_MockClient.md)
+ - [Mock a client for testing](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample7_MockClient.md)
```

Comment:
Do you think we should directly link to the MS Learn article here?  Doesn't seem to be helpful to jump to the old sample just to jump again.

Reply:
Yes, I'll add the link directly here.